source/installation/kafka-deployments/install-docker.txt

```diff
@@ -15,3 +15,77 @@ Install with a Containerized Cluster Using Docker
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+You can install Relational Migrator with Kafka in a containerized environment 
+using `Docker <https://docs.docker.com/engine/>`__. This method configures Kafka 
+to store data locally in Docker containers. Use this method if you want to learn 
+how to configure your own multi-server Kafka environment. 
+
+.. warning::
+
+   This deployment method is not recommended for production workloads. 
+
+About this Task
+---------------
+
+`Docker volumes <https://docs.docker.com/storage/volumes/>`__ can be used for 
```

Comment:
The compose file already includes Docker volumes, so we don't need to suggest this as an additional step. Instead it would be good to explain what's in the Compose file, specifically:
 - A Kafka node
 - A Kafka Connect node
 - A Relational Migrator node

Reply:
Updated!